doc/classes/Reference.xml

```diff
@@ -6,6 +6,7 @@
 	<description>
 		Base class for any object that keeps a reference count. [Resource] and many other helper objects inherit this class.
 		Unlike [Object]s, References keep an internal reference counter so that they are automatically released when no longer in use, and only then. References therefore do not need to be freed manually with [method Object.free].
+		If the extends keyword is omitted from the top of a script in GDScript, the class inherits from [Reference] by defualt.
```

Comment:
A typo, "default".

And the `extends` keyword should be formatted. With `[code]`, I think?

Reply:
Yeah - guess that's what I get for trying to multi-task like 4 different things...on two different computers. Fixed.